modules/custom/inope_policy_review/inope_policy_review.views.inc

```diff
@@ -62,12 +39,32 @@ function inope_policy_review_views_data() {
     ),
   );
 
-  // Policy Review NID
+  // Create the policy review id (primary key) views field.
+  $data['policy_review']['policy_review_id'] = array(
+    // The item it appears as on the UI.
+    'title' => t('Policy Review ID'),
+
+    // The help that appears on the UI.
+    'help' => t('The primary key.'),
```

Comment:
The primary key for...?  Might be useful - presumably there are other primary keys out there.  Just a thought.